CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ description: |-
 
 # VKCS Provider's changelog
 
+#### v0.1.12
+- Removed image_id attribute from resource_vkcs_images_image and hidden attribute from vkcs_images_image resource and data_source
```

Comment:
Could we split changes to image / floating_ip and tests to separate commits?

Reply:
Splitted